### DISCUSSION TITLE
Separator in choices?

### DISCUSSION THREAD

**Discussion by kentcdodds** [johnlindquist/kit#315] [Q&A]
With inquirer, I can have a separator: https://www.npmjs.com/package/inquirer#separator

Is this possible with Scriptkit?

**Comment by johnlindquist** [comment]
What's your use-case?

Since every prompt is an "autocomplete" prompt, I'm curious how separators would behave as the filtering kicks in.

Conceptually, `onTab` might be close to the concepts of grouping choices if that's what you're going for.

**Reply by kentcdodds** [reply]
Here's my use case:

![image](https://user-images.githubusercontent.com/1500684/121229269-f6e0e100-c84a-11eb-8cd3-53285c0711df.png)

I wanted a separator between the folders you can select and the actions you can take within the folder.
